sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TestUtils.java

```diff
@@ -324,21 +328,68 @@ static RecognizePiiEntitiesResultCollection getExpectedBatchPiiEntitiesForDomain
+        PiiEntityPropertiesHelper.setConfidenceScore(piiEntity0, 1.0);
+        PiiEntityPropertiesHelper.setOffset(piiEntity0, 0);
+
+        final PiiEntity piiEntity1 = new PiiEntity();
+        PiiEntityPropertiesHelper.setText(piiEntity1, "859-98-0987");
+        PiiEntityPropertiesHelper.setCategory(piiEntity1, PiiEntityCategory.USSOCIAL_SECURITY_NUMBER);
+        PiiEntityPropertiesHelper.setSubcategory(piiEntity1, null);
+        PiiEntityPropertiesHelper.setConfidenceScore(piiEntity1, 0.65);
+        PiiEntityPropertiesHelper.setOffset(piiEntity1, 28);
         return asList(piiEntity0, piiEntity1);
     }
 
     /**
      * Helper method to get the expected Categorized Entities List 2
      */
     static List<PiiEntity> getPiiEntitiesList2() {
-        // TODO: Use PiiEntityCategory after the class is introduced,
-        // issue: https://github.com/Azure/azure-sdk-for-java/issues/19180
-        PiiEntity piiEntity2 = new PiiEntity("111000025", EntityCategory.PHONE_NUMBER, null, 0.8, 18);
-        PiiEntity piiEntity3 = new PiiEntity("111000025", EntityCategory.fromString("ABARoutingNumber"), null, 0.75, 18);
-        PiiEntity piiEntity4 = new PiiEntity("111000025", EntityCategory.fromString("NZSocialWelfareNumber"), null, 0.65, 18);
-        return asList(piiEntity2, piiEntity3, piiEntity4);
+        String expectedText = "111000025";
+        final PiiEntity piiEntity0 = new PiiEntity();
+        PiiEntityPropertiesHelper.setText(piiEntity0, expectedText);
+        PiiEntityPropertiesHelper.setCategory(piiEntity0, PiiEntityCategory.PHONE_NUMBER);
+        PiiEntityPropertiesHelper.setSubcategory(piiEntity0, null);
+        PiiEntityPropertiesHelper.setConfidenceScore(piiEntity0, 0.8);
+        PiiEntityPropertiesHelper.setOffset(piiEntity0, 18);
+
+        final PiiEntity piiEntity1 = new PiiEntity();
+        PiiEntityPropertiesHelper.setText(piiEntity1, expectedText);
+        PiiEntityPropertiesHelper.setCategory(piiEntity1, PiiEntityCategory.ABAROUTING_NUMBER);
+        PiiEntityPropertiesHelper.setSubcategory(piiEntity1, null);
+        PiiEntityPropertiesHelper.setConfidenceScore(piiEntity1, 0.75);
+        PiiEntityPropertiesHelper.setOffset(piiEntity1, 18);
+
+        final PiiEntity piiEntity2 = new PiiEntity();
+        PiiEntityPropertiesHelper.setText(piiEntity2, expectedText);
+        PiiEntityPropertiesHelper.setCategory(piiEntity2, PiiEntityCategory.NZSOCIAL_WELFARE_NUMBER);
+        PiiEntityPropertiesHelper.setSubcategory(piiEntity2, null);
+        PiiEntityPropertiesHelper.setConfidenceScore(piiEntity2, 0.65);
+        PiiEntityPropertiesHelper.setOffset(piiEntity2, 18);
+
+        return asList(piiEntity0, piiEntity1, piiEntity2);
+    }
+
+    /**
+     * Helper method to get the expected batch of Personally Identifiable Information entities for categories filter
+     */
+    static RecognizePiiEntitiesResultCollection getExpectedBatchPiiEntitiesForCategoriesFilter() {
```

Comment:
could you add a test where you first call recognizepii with no filter, and then you use the categories from the output in the `categoriesFilter` input of a new call?
The result should be the same

Reply:
I can add one test scenarios for it.